### ISSUE TITLE
Signing the .dmg for MacOS

### ISSUE THREAD

**Issue by SimplyYourAverageDev** [qiuyuzhou/ShadowsocksX-NG#1495]
Sometimes when first-time users download the dmg file for MacOS, they have to go through a long process of allowing their Mac to run the application, would be great if we just signed it.

**Describe the solution you'd like**
We could try and sign the .dmg file for free if we register it with the [Fee Waiver Program](https://developer.apple.com/support/fee-waiver/) at apple.

**Comment by ghost** [comment]
Yeah I am feeling a bit like, how safe is this to use. I suppose I could check the checksum but man it feels weird to not have this signed and not be sure of what I am installing.
